### PR TITLE
docs(blog): speeding up your Program Indicators with Tracker-to-Aggre…

### PR DESCRIPTION
**Pull request by claudemamo**
…gate

### CODE REVIEW

**Review by germanviscuso — COMMENTED** [review]
Approved

File: blog/2022-02-10-automating-dhis2-integration-tests-in-junit-5.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 slug: 2022/02/automating-dhis2-integration-tests-in-junit-5
 title: Automating tests for DHIS2 integrations with JUnit 5
-author: claudemamo
```
Comment (amcgee) [inline blog/2022-02-10-automating-dhis2-integration-tests-in-junit-5.md:4]: You should instead add yourself as an author [here](https://github.com/dhis2/developer-portal/blob/fc9a5cea7a5809c3027ae0fa850581f64155a1cf/blog/authors.yml) and include the key to that collection as the `author` field

File: blog/2022-04-29-speeding-up-your-program-indicators-with-tracker-to-aggregate.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
+---
+slug: 2022/05/speeding-up-your-program-indicators-with-tracker-to-aggregate
+title: Speeding up your Program Indicators with Tracker-to-Aggregate
+author: Claude Mamo
```
Comment (amcgee) [inline blog/2022-04-29-speeding-up-your-program-indicators-with-tracker-to-aggregate.md:4]: Ref above

**Review by amcgee — CHANGES_REQUESTED** [review]
Will review the content later today, but added a minor point that jumped out at me and could be fixed quickly

**Pull request closed without merging**